test/consul.js

```diff
@@ -346,57 +346,6 @@ describe('Consul', () => {
     });
   });
 
-  it('resolves multiple tags as separate services', () => {
```

Comment:
I think I'll set this as a "pending" test. We can fill in the implementation when we include tag parsing.